packages/slice-machine/lib/mock/CustomType.ts

```diff
@@ -39,16 +41,23 @@ function buildDocumentMockConfig(
 
 export default function MockCustomType(
   model: CustomTypeSM,
-  legacyMockConfig: CustomTypeMockConfig
-) {
+  legacyMockConfig: CustomTypeMockConfig,
+  sharedSlices: Record<string, SharedSlice>
+): CustomTypeContent | null {
   const prismicModel = CustomTypes.fromSM(model);
   const documentMockConfig = buildDocumentMockConfig(
     prismicModel,
     legacyMockConfig
   );
-  return generateDocumentMock(
+
+  const mock = generateDocumentMock(
```

Comment:
can you use `DocumentMock.generate` so we don't use this helper that is no longer necessary since we don't require any renderer.
Also you can avoid the validator below, the content returned by `DocumentMock.generate` should be properly typed.
Also can you use `1.5.2` so we get a `Document` type instead of the previously named `CustomTypeContent` which is an odd name.